bridge/client/app/_components/ktb-variable-selector/ktb-variable-selector.component.html

```diff
@@ -0,0 +1,11 @@
+<button
+  dt-icon-button
+  variant="nested"
+  aria-label="Add secret"
```

Comment:
aria-label should be configurable, as we will set other things than secrets with it